create-release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Creates new package release in git.
+# a skip-merge is about not merge master to release.
+
+set -e
+set -o pipefail
+
+fail() {
+    echo "Error: $*" 1>&2
+    exit 1
+}
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
+
+if [[ "$current_branch" != "release" ]]; then
```

Comment:
I am guessing if you have different remotes, this will fail.
In my local env, I will use "git checkout release origin/release" to get the origin's release branch.
Maybe simply fail is better, let the user to checkout the correct release branch.

Reply:
Yes, correct, maybe just give an error info like original codes.
In my local env, I just want to fork a master in new folder, then all the things do automatically :).

Reply:
I add  codes to let ct choose if they want to checkout release automatically.